content/Foundation/Summit/2023/home.pml

```diff
@@ -34,34 +35,40 @@ Minneapolis, USA
     </h3>
   </div>
   <h3 class="highlight">
-    Register for the summit
+    Keynotes announced
   </h3>
   <p>
-    The seL4 summit will cover the complete seL4 ecosystem, consisting of the verified microkernel, as well as all seL4-related technology, tools, infrastructure, products, projects, and people.
+    We are pleased to announce two keynotes for the seL4 summit 2023.
   </p>
+  <div class="row" style="margin-left:5px; margin-right:5px">
   <p>
-    Tickets include:
+  <img src="/Foundation/Membership/LOGOS/NCSC.png"
+    style="width: 25%; float:right; margin-left:0px"
+    alt="NCSC logo" />
+    <strong>Gage</strong> is an Assurance Lead in NCSC with expertise in cryptography, software assurance and verification. He is part of a larger team that seeks to provide assurance and articulate risk for a wide range of products and customers. Whilst he is not a product developer himself, he has been responsible for presenting assurance cases for products that rely on seL4 to uphold security requirements.
   </p>
-  <ul>
-    <li>Participation in the 3-day conference, including talks, keynotes, seL4 updates & discussions</li>
-    <li>Networking with other seL4 experts and enthusiasts</li>
-    <li>Reception and Dinner</li>
-  </ul>
+  </div>
+  <div class="row" style="margin-left:5px; margin-right:5px">
   <p>
-    <strong><a href="https://events.linuxfoundation.org/sel4-summit/">Register here</a></strong>
+    <img src="../../../images/summit/sam-security-photo.jpg"
+    style="width: 15%; float:right; margin-left:100px"
+    alt="Sam Leffler" />
```

Comment:
For this summit page, I'd but both pictures (NCSC logo and Sam's picture) side by side centered (as we had before for Gernot and Boyd, or the panelists). It'll also avoid the impression Sam is from NCSC..